lib/rspec/rails/matchers/action_cable/have_broadcasted_to.rb

```diff
@@ -151,7 +160,7 @@ def data_description(data)
             if data.is_a?(RSpec::Matchers::Composable)
               data.description
             else
-              data
+              data.inspect
```

Comment:
What did this actually come from, I'm on the fence wether letting user values user `to_s` is fine or not.

Reply:
I would expect a matcher to use this if mentioning an object. The default description for the `eq` matcher does this:

```rb
it { expect("foo").to eq("foo") }
```

Resultant description:

```
is expected to eq "foo"
```

This adds some clarity to what the object is in context.